aws/v12.7.0/release.diff

```diff
@@ -0,0 +1,70 @@
+# Generated with:                                               <
+# devctl release create --name 12.6.0 --provider aws --base 12. <
```

Comment:
Why is this header gone? Was that changed in the tool or was the version used old?